scripts/svm/remoteHubPoolSetDepositRoute.ts

```diff
@@ -0,0 +1,298 @@
+// This script bridges remote call to pause deposits on Solana Spoke Pool. Required environment:
+// - ETHERS_PROVIDER_URL: Ethereum RPC provider URL.
+// - ETHERS_MNEMONIC: Mnemonic of the wallet that will sign the sending transaction on Ethereum
+// - HUB_POOL_ADDRESS: Hub Pool address
+
+import "dotenv/config";
+import * as anchor from "@coral-xyz/anchor";
+import { BN, Program, AnchorProvider, web3 } from "@coral-xyz/anchor";
+import { AccountMeta, PublicKey, SystemProgram } from "@solana/web3.js";
+import { SvmSpoke } from "../../target/types/svm_spoke";
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+import { ethers } from "ethers";
+import { MessageTransmitter } from "../../target/types/message_transmitter";
+import { decodeMessageHeader, getMessages } from "../../test/svm/cctpHelpers";
+import { HubPool__factory } from "../../typechain";
+import { ASSOCIATED_TOKEN_PROGRAM_ID, getAssociatedTokenAddressSync, TOKEN_PROGRAM_ID } from "@solana/spl-token";
+import {
+  CIRCLE_IRIS_API_URL_DEVNET,
+  CIRCLE_IRIS_API_URL_MAINNET,
+  SOLANA_USDC_DEVNET,
+  SOLANA_USDC_MAINNET,
+} from "./utils/constants";
+import { fromBase58ToBytes32, fromBytes32ToAddress } from "./utils/helpers";
+
+// Set up Solana provider.
+const provider = AnchorProvider.env();
+anchor.setProvider(provider);
+
+// Parse arguments
+const argv = yargs(hideBin(process.argv))
+  .option("seed", { type: "string", demandOption: true, describe: "Seed for the state account PDA" })
```

Comment:
this might be redundant now or we still expect to deploy test builds that allow non-zero seeds?